### PR TITLE
release-4.1.0 prepare release

### PR DESCRIPTION
**Pull request by marcin-onegini**

### CODE REVIEW

File: src/ios/build.gradle [hunk 1]
```diff
@@ -54,11 +54,11 @@ configurations {
 
 // TODO Move to release version before plugin release
```
Comment (steinwelberg) [inline src/ios/build.gradle]: Please remove the TODO.

File: docs/public/release-notes/4.x.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Release notes 4.x
 
+## 4.1.0
+### Features
+- Added pending mobile authentication push requests
+
+### Improvements
+- [`Android`] Updated third party libraries to the latest versions.
```
Comment (steinwelberg) [inline docs/public/release-notes/4.x.md]: This is a bit cryptic and doesn't really tell something. Either we should mention which third party libraries we have update or we shouldn't mention this. 

I guess this is about the native dependencies, right? In that case I wonder whether we should mention it at all..

Comment (marcin-onegini) [reply docs/public/release-notes/4.x.md]: yes, it's about google-play-services and firebase. I will remove it.

File: src/ios/build.gradle (hunk 1 above)
Comment (marcin-onegini) [reply src/ios/build.gradle]: done

**Pull request merged**